packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker/create-transform-job.ts

```diff
@@ -173,6 +181,23 @@ export class SageMakerCreateTransformJob extends sfn.TaskStateBase {
     };
   }
 
+  private renderModelClientOptions(config: ModelClientOptions): { [key: string]: any } {
+    const retries = config.invocationsMaxRetries;
+    if (retries? (retries < 0 || retries > 3): false) {
+      throw new RangeError(`invocationsMaxRetries should be between 0 and 3, not ${retries}.`);
```

Comment:
```suggestion
      throw new RangeError(`invocationsMaxRetries should be between 0 and 3. Received: ${retries}.`);
```

Reply:
Done. Didn't realize it was not being used anymore since I saw it in a couple places. Good recommendation, though!